config/locales/en.yml

```diff
@@ -46,6 +46,7 @@ en:
         submit: 'Go'
   hyku:
     splash:
+      account_login: Login to get started
```

Comment:
We'll need to i18nize these once the PR is ready. I would be happy to do this after we merge, though you may need to remind me!